readme.md

```diff
@@ -1,2 +1,2 @@
-## Hanged Man Game
+## Hanged Man hangedMan.logic.Game
```

Comment:
I guess this is not intentional, to put the fully qualified class name to readme file header

Reply:
Yes, it's not. No idea, why that happened... But when I changed the name of the package, it was also changed in readme.md. I erased it nevetheless.